docs/tutorials/mongoid-documents.txt

```diff
@@ -176,6 +177,31 @@ Types that are not supported as dynamic attributes since they cannot be cast are
 - ``DateTime``
 - ``Range``
 
+The StringifiedSymbol Type
+--------------------------
+
+This type allows users to write Ruby code that makes use of the built-in Ruby Symbol type,
+while ensuring interoperability with other drivers. The StringifiedSymbol type stores all data
+on the MongoDB backend as a String, while returning values to the user as a Symbol.
```

Comment:
I don't think we use the term "MongoDB backend" -- I would use "the database" instead.